amethyst/src/main/java/com/vitorpamplona/amethyst/ui/components/ZoomableContentView.kt

```diff
@@ -708,6 +714,7 @@ fun ShareImageAction(
     hash: String?,
     mimeType: String?,
     onDismiss: () -> Unit,
+    content: BaseMediaContent? = null,
 ) {
```

Comment:
You can put the coroutine scope here.
```suggestion
) {

   val scope = rememberCoroutineScope()
```